cat-boot-report-pdf/src/test/java/cc/catalysts/boot/report/pdf/impl/DemoTest.java

```diff
@@ -55,7 +55,8 @@ public void demo() throws Exception {
 
 
         final File target = new File("pdf-out");
-        Assert.assertTrue(target.mkdirs());
+        target.mkdirs();
+        Assert.assertTrue(target.exists());
```

Comment:
I would prefer a solution where the intention is a little bit easier to see, something like:
```
if (!target.exists()) {
    Assert.assertTrue(target.mkdirs());
}
```

and in theory we would also have to ensure that it actually is a directory